pkg/client/supplier/client.go

```diff
@@ -15,6 +15,15 @@ import (
 
 var _ client.SupplierClient = (*supplierClient)(nil)
 
+const (
+	// TODO_TECHDEBT(@okdas): Gas limit and price should have their dedicated configuration entry.
+	// gasLimit is the default gas limit of the relay miner transactions.
+	gasLimit = 100000
```

Comment:
```suggestion
	gasLimit = 300000
```
Let's make it larger and see if it'll be enough.